core/server/api/v2/utils/serializers/output/utils/mapper.js

```diff
@@ -11,6 +12,21 @@ const mapPost = (model, frame) => {
     if (utils.isContentAPI(frame)) {
         date.forPost(jsonModel);
         members.forPost(jsonModel, frame);
+
+        const excerpt = require('../../../../../../data/meta/excerpt');
```

Comment:
IMO we should not be depending on the data/meta folder inside the API

- data/meta is SDK layer, the SDK layer should depend on the API, not the other way around.
- data/meta/excerpt is the _old_ version of the excerpt which we are replacing with the spec in #10062

The new spec is:  return either the custom excerpt, or the first 500 words of the plaintext version of the content.

The aim is to be able to remove the excerpt calculations from elsewhere in the codebase, and eventually get rid of our dependency on downsize, which produces broken, ugly excerpts.

Reply:
> IMO we should not be depending on the data/meta folder inside the API

Yeah 100%. This was just me finishing the PR so it is def mergable for the release, because it works. PR is still in WIP. Priority is to finish the required issues.

We don't have access to the plaintext. By default we serve `html` only.

We could put a tiny helper fn for now into the serializer and just remove the html tags and take a substring of the content (0, 500 chars). If that's good enough for now 👍

Reply:
That wouldn't comply with the spec. We specifically want to use the plaintext field. Not really understanding why this is difficult 🧐